bedrock/base/templates/base-new.html

```diff
@@ -7,13 +7,13 @@
   <head>
     <meta charset="utf-8">{# Note: Must be within first 512 bytes of page #}
 
+    {% block experiments %}{% endblock %}
```

Comment:
This is an excellent point. I think the lib should respect DNT and halt initialization if DNT is enabled. Will update.